src/lib/util.js

```diff
@@ -742,8 +741,9 @@ export function handleRepresentationBlobFetch(response) {
  * Useful for determining the true state of a "successful" representation
  *
  * @param {string} contentUrl - a representation's authenticated content URL
+ * @param {Axios} api - Instance of API layer
  * @return {Promise} - Response from axios
  */
-export function fetchRepresentationAsBlob(contentUrl) {
+export function fetchRepresentationAsBlob(contentUrl, api) {
```

Comment:
This seems a little janky. I think this helper is only used in one place. Maybe it'd be simpler to inline it where it's used?